site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -289,5 +278,37 @@ if it isn't there already.
   located in the `my-project` directory,
   letting it import other modules from that directory.
   ```bash
-  PYTHONPATH=path/to/my-project cvat-cli auto-annotate 139 --function-module my_func
+  PYTHONPATH=path/to/my-project cvat-cli task auto-annotate 139 --function-module my_func
+  ```
+
+## Examples - projects
+
+### Create
+
+To create a project, you have to define its labels.
```

Comment:
Actually, labels are optional for projects. They can be initialized on dataset import.

Reply:
Yeah, I don't know why I thought they were mandatory. Updated.

(Also, while updating, I realized that setting the labels doesn't even work. :S Fixed that too.)